src/tools/rados/rados.cc

```diff
@@ -1921,10 +1936,23 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
       goto out;
     }
     if (!formatter) {
-      printf("  total used    %12lld %12lld\n", (long long unsigned)tstats.kb_used,
-	     (long long unsigned)tstats.num_objects);
-      printf("  total avail   %12lld\n", (long long unsigned)tstats.kb_avail);
-      printf("  total space   %12lld\n", (long long unsigned)tstats.kb);
+      cout << std::endl;
+      cout << "=============================================================="
+           << std::endl;
+      cout << "Total Cluster Usage"
+           << std::endl;
+      cout << "=============================================================="
+           << std::endl;
+      cout << "total_objects    " << tstats.num_objects
+           << std::endl;
+      cout << "total_used       " << si_t(tstats.kb_used << 10)
+           << std::endl;
+      cout << "total_avail      " << si_t(tstats.kb_avail << 10)
+           << std::endl;
+      cout << "total_space      " << si_t(tstats.kb << 10)
+           << std::endl;
+      cout << "=============================================================="
+           << std::endl;
```

Comment:
For this last section, can we drop tne ==== lines (a blank line is sufficient), and the _'s?